python/cudf/cudf/utils/ioutils.py

```diff
@@ -1536,11 +1539,13 @@ def is_directory(path_or_data, storage_options=None):
     return False
 
 
-def _get_filesystem_and_paths(path_or_data, storage_options):
+def _get_filesystem_and_paths(path_or_data, storage_options, filesystem=None):
```

Comment:
```suggestion
def _get_filesystem_and_paths(path_or_data, storage_options, *, filesystem=None):
```

And matching changes elsewhere to always use the keyword form.